drafts/use-cases/2.api-documentation.md

```diff
@@ -0,0 +1,111 @@
+## API Documentation
+
+### Story
+As an application user
+I want my application to be able to perform some precomputations 
+So it can pre-generate some views and other UI elements to speed things up.
```

Comment:
Another story might be

As a API consumer
I want to be able to get an overview of the structure and capabilities of a Hydra API
So that I understand how to interact with the API and make those interactions more efficient.